orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/operators/grammatical/IfDecisionBased.feature

```diff
@@ -0,0 +1,3900 @@
+Feature: Values can be specified by using if, then and else constraints
```

Comment:
Ideally we would have some sort of syntax to run the tests for each walker. If that is infeasible, perhaps a comment should be added to the top of this file (and the one it is copied from) so that someone adding/removing tests in future will be more likely to repeat for the duplicate file.